doc/migration/version-upgrade-guide.md

```diff
@@ -972,6 +1028,19 @@ With:
 ```yaml
 - name: {{your operator name which is the value of metadata.name in this file}}
 ```
+
+By default the full Ansible logs will not be output, however, you can setup it via the `ANSIBLE_DEBUG_LOGS` environment variable in the `deploy/operator.yaml` file. See:
+
+```
+...
+- name: ANSIBLE_DEBUG_LOGS
+  value: "True"
+...
+```
+
+**NOTE:** The dependency `inotify-tools` on Ansible based-operator images has been deprecated since its usage was required just because of this container sidecar. It will be removed for the next versions. 
+- Now, you can use the environmentt 
```

Comment:
Is the `inotify-tools` change user-facing? If not, we should not put it in the migration guide since there is nothing for a user to do.